kibbeh/src/ui/mobile/AccountOverlay.tsx

```diff
@@ -79,15 +79,15 @@ export const AccountOverlay: React.FC<AccountOverlyProps> = ({}) => {
   return createPortal(
     <a.div className="absolute w-screen h-full" style={{ display }}>
       <a.div
-        className="w-screen h-screen absolute top-0 left-0 bg-black z-10 opacity-100"
+        className="w-screen h-screen absolute left-0 bg-black z-10 opacity-100"
         onClick={() => close()}
         style={bgStyle}
       ></a.div>
       <a.div
         className="bg-primary-800 w-full h-full rounded-t-20 relative pt-5"
         {...bind()}
         style={{
-          bottom: `calc(-100% + ${height - 100}px)`,
+          bottom: `calc(-100% + ${height - 70}px)`,
```

Comment:
```suggestion
          bottom: `calc(-100% + ${height - 100}px)`,
```

The margin below logout was intentional. If it's an electron thing, make it electron exclusive

Reply:
done